packages/advanced-logic/test/extensions/payment-network/ethereum/fee-proxy-contract.test.ts

```diff
@@ -595,4 +629,43 @@ describe('extensions/payment-network/ethereum/fee-proxy-contract', () => {
       }).toThrowError('feeAmount is not a valid amount');
     });
   });
+
+  describe('applyActionToExtension/declarePayment', () => {
```

Comment:
Missing: It should not crash if the amount is not valid on applyDeclarePaymentAction